docs/source/_static/custom.js

```diff
@@ -0,0 +1,26 @@
+document.addEventListener("DOMContentLoaded", function () {
+  var script = document.createElement("script");
+  script.type = "module";
+  script.id = "runllm-widget-script"
+
+  script.src = "https://widget.runllm.com";
+
+  script.setAttribute("version", "stable");
+  script.setAttribute("runllm-keyboard-shortcut", "Mod+j"); // cmd-j or ctrl-j to open the widget.
+  script.setAttribute("runllm-name", "DaskBot");
+  script.setAttribute("runllm-position", "BOTTOM_RIGHT"); // put above ethical ads
+  script.setAttribute("runllm-position-x", "20px");
+  script.setAttribute("runllm-position-y", "50%");
+  script.setAttribute("runllm-assistant-id", "273");
+  script.setAttribute("runllm-theme-color", "#FFC11E");
+  script.setAttribute("runllm-slack-community-url", "https://dask.slack.com/");
+  script.setAttribute("runllm-per-user-usage-limit", 2);
+  script.setAttribute("runllm-usage-limit-effective-days", 30);
+  script.setAttribute("runllm-usage-limit-message", `Hi! You've hit the limit for anonymous questions, but you can join us on Slack in #DaskBot and ask as many questions as you'd like.`);
```

Comment:
Slight rewording of the message folks get when the hit the limit

```suggestion
  script.setAttribute("runllm-usage-limit-message", `Hi! You've hit the limit for questions here in the docs, but you can join us on the Dask community Slack and ask as many questions as you'd like in #DaskBot channel. Hope to see you there.`);
```